nixos/modules/services/x11/window-managers/clfswm.nix

```diff
@@ -8,17 +8,28 @@ in
 
 {
   options = {
-    services.xserver.windowManager.clfswm.enable = mkEnableOption "clfswm";
+    services.xserver.windowManager.clfswm = {
+      enable = mkEnableOption "clfswm";
+      package = mkOption {
+        type        = types.package;
+        default     = pkgs.lispPackages.clfswm;
+        defaultText = "pkgs.lispPackages.clfswm";
+        example     = "pkgs.lispPackages.clfswm";
```

Comment:
`example` isn't overly useful. Can you put something about custom packages in the `description`, if appropriate? Other than that, LGTM :+1:

Reply:
Fixed it, thanks for the reminder. But it doesn't seem necessary, as it can be done with `overlay` (unless someone needs both versions).

Reply:
Hmm... yeah that `example` is a bit verbose. Maybe just drop the `example` field. I think anyone wanting to use it probably knows how already... ?